src/main/java/TestFile2.java

```diff
@@ -0,0 +1,16 @@
+class TestFile2 {
+    public String foo() {
+        return "bar";
+    }
+
+    private static void incorrectAssignmentInIfCondition() {
+        boolean value = false;
+        if (value == true) {
+            //do Something
+            value = false;
+        }
+        if (value == false) {
+          int x = 1;
+        }
+    }
```

Comment:
[Checkstyle] WARNING: 'method def rcurly' have incorrect indentation level 4, expected level should be 2.